subprojects/core-api/src/main/java/org/gradle/work/FileChange.java

```diff
@@ -37,4 +37,17 @@
      * The type of change to the file.
      */
     ChangeType getChangeType();
+
+    /**
+     * The normalized path of the file, as specified by the path normalization strategy.
+     *
+     * <p>
+     *    Examples:
+     * </p>
+     * <ul>
+     *     <li>For {@link org.gradle.api.tasks.PathSensitivity#NAME_ONLY} this is the file name.</li>
+     *     <li>For {@literal @}{@link org.gradle.api.tasks.Classpath} this is empty for Jar files and the relative path (i.e. package name) of the class files.</li>
```

Comment:
Is it the package name, or is it `packageName.replaceAll('.', '/') + name`?

Reply:
We should also mention `RELATIVE`, probably as the first item given its the most frequently used sensitivity.

Reply:
I think it would be better to improve the documentation in `PathSensitivity` instead of having the information only here. It seems like `PathSensitivity.RELATIVE` is not very often understood. WDYT?

Reply:
I added more documentation to `PathSensitivity.RELATIVE` and reference that documentation from here.